src/SelectField/Select.js

```diff
@@ -99,8 +100,9 @@ export class Select extends Component {
 }
 
 Select.propTypes = {
-    value: propTypes.string,
+    name: propTypes.string.isRequired,
     onChange: propTypes.func.isRequired,
+    value: propTypes.string,
```

Comment:
This could be made required. Empty string would suffice that condition.
But I don't care too much about it